nautobot/core/templates/generic/object_bulk_update.html

```diff
@@ -27,14 +27,16 @@ <h1>{% block title %}Editing {{ table.rows|length }} {{ obj_type_plural|betterti
         {{ field }}
     {% endfor %}
     <div class="row">
+        {% if table %}
         <div class="col-md-8">
             <div class="panel panel-default">
                 <div class="table-responsive">
                     {% render_table table 'inc/table.html' %}
                 </div>
             </div>
         </div>
-        <div class="col-md-4">
+        {% endif %}
+        <div class="{% if edit_all %} col-lg-8 col-lg-offset-2 col-md-10 col-md-offset-1 {% else %} col-md-4 {% endif %}">
```

Comment:
Do we really need a separate `edit_all` flag in this template or could this just be changed to `if not table`?